elasticsearch/init/0001-jaeger

```diff
@@ -22,7 +22,6 @@ info "Starting init script: ${script}"
 
 if [ -n "${NAMESPACE:-}" ] ; then
  sed -i -e 's/namespace:.*/namespace: '"$NAMESPACE"'/' ${HOME}/sgconfig/sg_config.yml
- sed -i -e 's/user.jaeger/user.'"$NAMESPACE"'.jaeger/' ${HOME}/sgconfig/sg_roles_mapping.yml
```

Comment:
I have talked with @ewolinetz about this. We don't have to change the `CN` because every jaeger instance has different CA and key. 

Our deployment:
Each jaeger instance has unique elasticsearch cluster. The operator creates CA and all the certs for each jaeger instance individually. 

cc @jpkrohling